broker/cloud_run/lsst/lite/main.py

```diff
@@ -0,0 +1,199 @@
+    )
+    # create the lite dictionary for the outgoing alert
+    alert_lite_dict = {
+        alert.get_key("alertid"): alert.get("alertid"),
+        alert.get_key("source"): source_lite_dict,
+        alert.get_key("prv_sources"): prev_sources_lite_dict,
+        alert.get_key("prv_forced_sources"): alert.dict.get("prvDiaForcedSources"),
+        alert.get_key("prv_nondetect_limits"): alert.dict.get("prvDiaNondetectionLimits"),
+        alert.get_key("object"): object_lite_dict,
+        alert.get_key("ss_object"): ssobject_lite_dict,
+    }
+
+    return pittgoogle.Alert.from_dict(payload=alert_lite_dict, schema_name=f"{SURVEY}")
+
+
+def _get_fields(field_name: str) -> list[str]:
+    """Returns a list of survey-specific field names that will be included in a lite dictionary."""
+    if field_name == "diaSource":
+        source_fields_list = [
+            "diaSourceId",
+            "midpointMjdTai",
+            "ra",
+            "raErr",
+            "dec",
+            "decErr",
+            "psfFlux",
+            "psfFluxErr",
+            "band",
+        ]
+
+        return source_fields_list
+
+    if field_name == "diaObject":
+        object_fields_list = [
+            "diaObjectId",
+            "nearbyObj1",
+            "nearbyObj2",
+            "nearbyObj3",
+            "nearbyObj1Dist",
+            "nearbyObj2Dist",
+            "nearbyObj3Dist",
+            "nearbyObj1LnP",
+            "nearbyObj2LnP",
+            "nearbyObj3LnP",
+            "u_psfFluxErrMean",
+            "g_psfFluxErrMean",
+            "r_psfFluxErrMean",
+            "i_psfFluxErrMean",
+            "z_psfFluxErrMean",
+            "y_psfFluxErrMean",
+        ]
+
+        return object_fields_list
+
+    if field_name == "ssObject":
+        ss_object_fields_list = [
+            "ssObjectId",
+            "firstObservationDate",
+            "arc",
+            "numObs",
+            "MOID",
+            "MOIDEclipticLongitude",
+            "MOIDDeltaV",
+            "u_H",
+            "u_HErr",
+            "u_Chi2",
+            "g_H",
+            "g_HErr",
+            "g_Chi2",
+            "r_H",
+            "r_HErr",
+            "r_Chi2",
+            "i_H",
+            "i_HErr",
+            "i_Chi2",
+            "z_H",
+            "z_HErr",
+            "z_Chi2",
+            "y_H",
+            "y_HErr",
+            "y_Chi2",
+            "medianExtendedness",
+        ]
+
+        return ss_object_fields_list
+
+    raise ValueError(
+        f"Unrecognized field type: {field_name}. Only 'diaSource', 'diaObject', and 'ssObject' are supported."
+    )
+
+
+def _create_lite_dict(alert_dict: dict, field_names: list[str]) -> dict:
+    """Returns a lite dictionary containing fields specified in field_names."""
+    if alert_dict is None:
+        return alert_dict
+    return {k: v for k, v in alert_dict.items() if k in field_names}
+
+
+def _create_prv_sources_lite_dict(
```

Comment:
@troyraen Codacy had issues with the `|` operand:
```
unsupported operand type(s) for |
```
I've updated these lines to address the issue.

Reply:
Using `typing.Optional` is fine. It shouldn't be necessary as of python 3.9(?) but sometimes it can be tricky to get the syntax right with `|`.